lib/paths.cf

```diff
@@ -315,7 +315,7 @@ bundle common paths
       "path[tr]"            string => "/usr/bin/tr";
       "path[wc]"            string => "/usr/bin/wc";
       "path[wget]"          string => "/usr/bin/wget";
-      "path[realpath]"      string => "/usr/bin/realpath";
+      "path[realpath]"      string => "/usr/bin/readlink -e";
```

Comment:
> Still it makes me slightly uncomfortable that we are kind of lying about the path to the binary. Its not really realpath, its really readlink.

The problem is that in older versions of coreutils realpath was a binary on redhat as well as several other platforms as well. If you actually make use of $(paths.path[realpath]) across multiple platforms you will now have to rewrite these agents and jump through hoops.

Seems like changing this to readlink will offer backwards compatibility